cmd/notifier/config/config.toml

```diff
@@ -8,3 +8,4 @@ subscribed_addresses = ["erd1", "erd1"]
     retry_duration = 5
     # Signals if in case of data payload processing error, we should send the ack signal or not
     blocking-ack-on-error = false
+    blocks-cache-size = 100
```

Comment:
add a comment to keep the file consistent

Reply:
This part was removed since we added Extract and removed the lru part from cache